Sources/TokamakCore/MountedComponents/MountedView.swift

```diff
@@ -36,15 +36,15 @@ public class MountedView<R: Renderer> {
 }
 
 extension View {
-  func makeMountedView<R: Renderer>(_ parentTarget: R.TargetType)
+  func makeMountedView<R: Renderer>(_ parentTarget: R.TargetType, withEnvironment environmentValues: EnvironmentValues)
```

Comment:
**suggestion:** as `EnvironmentValues` is a unique type (highly unlikely you'll pass something other that the correct value of that type), I don't think the keyword argument is necessary here, it will also make a lot of function applications shorter elsewhere

```suggestion
  func makeMountedView<R: Renderer>(_ parentTarget: R.TargetType, _ environmentValues: EnvironmentValues)
```